Android.html

```diff
@@ -0,0 +1,126 @@
+LINK TO THE EDITTED ANDROID: file:///Users/marcovienna/2015week1/Android.html
```

Comment:
This link doesn't work (try a hyperlink in Github markdown!, but the code works perfectly. Good work!